pkg/webui/console/containers/device-onboarding-form/device-type-form-section/device-type-manual-form-section/validation-schema.js

```diff
@@ -108,7 +108,7 @@ const macSettingsSchema = Yup.object({
           class_c_timeout: Yup.string(),
         }
 
-        // Each MAC setting that does have a corresponding default setting is required.
+        // Each MAC setting that does not have a corresponding default setting is required.
```

Comment:
```suggestion
        // Each MAC setting that does have a corresponding default setting is required.
```
Sorry for the confusion, but this was actually right. What I mean here is that any value that has a default should not be cleared, since it will be prefilled with the default and I want to make the settings as explicit as possible. Note that this does not mean that all fields that do not have a default setting are automatically not required. Eg. the ping slot periodicity has no default value but needs to be provided explicitly by the user and these exceptions are defined explicitly in the validation schema.